### DISCUSSION TITLE
I CAN'T LOG IN TO WINDOWS AFTER RUNNING THE SCRIPT

### DISCUSSION THREAD

**Discussion by zeyrouc** [simeononsecurity/Windows-Optimize-Harden-Debloat#101] [General]
I can't access the windows hop I did the steps before running the script but it seems that I didn't understand well, it's not that you could give me a tutorial on how to create the account

**Comment by simeononsecurity** [comment]
Be sure you read the readme before running random scripts online
https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat#fixing-microsoft-account-store-or-xbox-services

If you can't login anymore, time to reinstall windows or attempt to get into the boot menu too reset to a snapshot.
https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/wiki/How-to-Reverse-the-Changes-performed-in-Windows%E2%80%90Optimize%E2%80%90Harden%E2%80%90Debloat

**Comment by zeyrouc** [comment]
Then copy the contents of the previous users folder into the new one after signing into the new user for the first time to work around this prior to running the script.
I don't understand what I have to copy to which folder and this is the difficult part I don't understand how to do

**Reply by simeononsecurity** [reply]
> Then copy the contents of the previous users folder into the new one after signing into the new user for the first time to work around this prior to running the script. I don't understand what I have to copy to which folder and this is the difficult part I don't understand how to do

Hi @zeyrouc,

I understand how this part can be confusing. Here are the steps in more detail:

1. **Create a New User Account:** If you're locked out of your current account, first create a new local user account by accessing the boot menu and selecting a reset option that allows you to create a new user. You can find help on creating a new user account [here](https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/wiki/How-to-Reverse-the-Changes-performed-in-Windows%E2%80%90Optimize%E2%80%90Harden%E2%80%90Debloat).

2. **Sign In to the New User Account:** Once you have created the new account, sign in to Windows with this new account for the first time. This will generate the necessary user folders for this new account.

3. **Copy Contents from the Old Account:** You need to transfer your personal files from the old user account folder. To do this:
   - Go to the old user's folder, usually located in `C:\Users\OldUsername`.
   - Copy the contents of this folder (like Documents, Desktop, Pictures, etc.) to the corresponding folders in your new user account (`C:\Users\NewUsername`). This way, you restore your personal files and settings to the new account.

Make sure you don't copy any hidden system files that might carry over unwanted configurations. Focus on your personal files.

If you need further help, feel free to ask!
